src/collectors/windows.plugin/metadata.yaml

```diff
@@ -274,3 +274,150 @@ modules:
               chart_type: line
               dimensions:
                 - name: temperature
+  - meta:
+      plugin_name: windows.plugin
+      module_name: PerflibWebService
+      monitored_instance:
+        name: IIS
+        link: "https://www.iis.net/"
+        categories:
+          - data-collection.windows-systems
+        icon_filename: "windows.svg"
+      related_resources:
+        integrations:
+          list: []
+      info_provided_to_referring_integrations:
+        description: ""
+      keywords:
+        - IIS
+        - HTTP
+        - Web service
+      most_popular: false
+    overview:
+      data_collection:
+        metrics_description: |
+          This collector monitors the current number of processes, threads, and context switches on Windows systems.
```

Comment:
I believe this is copy-pasted and missed, please change to proper sentence

Reply:
You are right, I am sorry for this mistake. I pushed a commit fixing it.